compiler/ballerina-lang/src/main/resources/grammar/BallerinaLexer.g4

```diff
@@ -33,6 +33,7 @@ RETURNS     : 'returns';
 VERSION     : 'version';
 DOCUMENTATION  : 'documentation';
 DEPRECATED  :  'deprecated';
+RECORD      : 'record';
```

Comment:
Shall we move this near to Object token. (Line24)

Reply:
Addressed with commit 3344b24